middleware/helpers.ts

```diff
@@ -35,7 +35,10 @@ export function createDurationTimer(req: IncomingMessage): EndTimer {
   return end;
 }
 
-export function addressToString(addr: string | AddressInfo) {
+export function addressToString(addr: string | AddressInfo | null) {
+  if (addr === null) {
+    return "";
+  }
```

Comment:
Is this okay to do? Null for `server.address()` introduced in `@types/node@12.0.0`

<img width="566" alt="image" src="https://user-images.githubusercontent.com/7439294/227853543-2ec708a9-7e39-458d-b339-3bb0feeb5669.png">

I tried using an if statement before calling this method and it didn't work.

Reply:
We are only calling this in the on("listening") event so it should never be null but using an `as string | AddressInfo` earlier feels a little "smelly"

Reply:
Not sure what it is used for? Just for display purposes?
If so I wonder if `"unknown"` rather than `""` is better - doesn't become invisible.